src/ProbeMeasurements.hpp

```diff
@@ -0,0 +1,53 @@
+#ifndef WATER_PROBE_ACQUANATIVA_AP3_PROBE_MEASUREMENTS_HPP
+#define WATER_PROBE_ACQUANATIVA_AP3_PROBE_MEASUREMENTS_HPP
+
+#include <base/Float.hpp>
+#include <base/Temperature.hpp>
+#include <base/Time.hpp>
+
+namespace water_probe_acquanativa_ap3 {
+    struct ProbeMeasurements {
+        base::Time time;
+        
+        /** Water dissolved oxygen concentration in Kg/L */
+        float concentration = base::unknown<float>();
+
+        /** Water dissolved oxygen % saturation */
```

Comment:
Which is of course represented e.g. 0.1 for 10%

Reply:
oh, I was considering 10%